### ISSUE TITLE
stac-browser always shows the default catalog

### ISSUE THREAD

**Issue by JamesTownend** [radiantearth/stac-browser#76]
Running the following

```
npm install
CATALOG_URL=https://landsat-stac.s3.amazonaws.com/catalog.json npm start -- --open
```

or the same with other working STAC catalogs always produces a stac-browser webpage for the default Planet Disaster Data.

I'm using WSL2 on Windows 10, which may or may not be part of the issue.

**Comment by m-mohr** [comment]
Interesting, thanks for reporting. I'm never using env variables (only the config file instead) that's why I did not find out that env variables got broken with the vue-cli migration. I need to figure out tomorrow how to solve this, until then please use the config file.

Note to self: https://forum.vuejs.org/t/how-to-pass-command-line-parameters-to-vue/38701/13

**Comment by JamesTownend** [comment]
That workaround works. Thanks

**Comment by m-mohr** [comment]
Just fixed it and released as v2.0.0-rc.3. Please verify that it works for you.

Please note that env variables have been deprecated and replaced with CLI parameters, so for example your start command above would now translate into
`npm start -- --open --CATALOG_URL="https://landsat-stac.s3.amazonaws.com/catalog.json"`

That makes things work more reliable across OS.

**Comment by JamesTownend** [comment]
It works for me. Thanks
